[ #14 ] 마이페이지 조회, 비밀번호 수정, 이미지 수정, 회원가입, 로그인, 토큰 재발급 테스트코드 작성

suyeonkim56 opened the pull request:
#14 

마이페이지 조회, 비밀번호 수정, 이미지 수정, 회원가입, 로그인, 토큰 재발급 테스트코드 작성

suyeonkim56 committed:
Merge branch 'dev' into feat/login

suyeonkim56 committed:
Merge branch 'dev' into feat/login

suyeonkim56 committed:
Test : 마이페이지 조회 testcode 작성 ( #14 )

suyeonkim56 committed:
Merge branch 'dev' into feat/login

suyeonkim56 committed:
Test : 비밀번호 수정, 이미지 수정 testcode 작성 ( #14 )

suyeonkim56 committed:
Test : 회원가입, 로그인 테스트코드 작성 ( #14 )

suyeonkim56 committed:
Test : 토큰 재발급 서비스 testcode 작성 ( #14 )

suyeonkim56 committed:
Merge branch 'dev' into feat/login

suyeonkim56 committed:
Test : 토큰 재발급 컨트롤러 testcode 작성 ( #14 )

suyeonkim56 committed:
Chore : 안쓰는 import 구문 삭제 ( #14 )

suyeonkim56 committed:
Chore : 안쓰는 import 구문 삭제 ( #14 )

seiky17 reviewed (approved):
깔끔하게 잘 작성하셨네요! 고생하셨습니다!

kyn1013 commented on src/test/java/com/example/docconneting/domain/auth/controller/AuthControllerTest.java:
```diff
@@ -0,0 +1,119 @@
+        ReflectionTestUtils.setField(request, "refreshToken", "refresh");
+
+        String newAccessToken = "new";
+        String newRefreshToken = "new";
```
new 보다는 엑세스 토큰 리프레시 토큰 둘이 서로 다른 문자열이면 더 좋을 것 같습니다 ㅎㅎ

kyn1013 commented on src/test/java/com/example/docconneting/domain/auth/service/AuthServiceTest.java:
```diff
@@ -0,0 +1,351 @@
+        Long ttl = 100000L;
+
+        String newAccessToken = "new";
+        String newRefreshToken = "new";
```
여기도 둘이 문자열만 다르면 좋을 것 같습니당

kyn1013 commented on src/test/java/com/example/docconneting/domain/auth/service/AuthServiceTest.java:
```diff
@@ -0,0 +1,351 @@
+        Long ttl = 100000L;
+
+        String newAccessToken = "new";
+        String newRefreshToken = "new";
```
여기도요!

tggKim commented on src/main/java/com/example/docconneting/domain/user/service/UserService.java:
```diff
@@ -67,6 +70,7 @@ public Map<String, String> updateImage(AuthUser authUser, UpdateImageRequest dto
         user.updateImage(dto.getNewImage());
         Map<String, String> message = new HashMap<>();
         message.put("message","이미지 수정이 성공적으로 됐습니다");
+        entityManager.flush();
```
여기에서는 수정날짜를 반영하는 dto가 아니어서 굳이 flush()를 하지 않아도 될 것 같습니다

kyn1013 reviewed (approved):
저는 이 부분만 고치시면 좋을 것 같습니다 코드 짜시느라 완전 수고하셨습니다 !!

wonchaebin commented on src/test/java/com/example/docconneting/domain/auth/service/AuthServiceTest.java:
```diff
@@ -0,0 +1,351 @@
+        Map<String, String> response = authService.signUp(request);
+
+        //then
+        assertEquals("회원 가입이 성공적으로 됐습니다", response.get("message"));
```
테스트에서 assertThat과 assertEquals가 혼용되어있는데 통일성을 위해 형식을 맞추는 건 어떨지 여쭤보고 싶습니다!
assertThat이 가독성 측면에서는 더 좋다고 하더라구요!

wonchaebin reviewed (approved):
테스트 코드 짜시느라 고생 많으셨습니다!

suyeonkim56 committed:
Chore : 피드백 기반 리팩토링 ( #14 )

- assertEqual문 assertThat으로 변경

- newAccessToken, newRefreshToken 실제 부여 값 불일치하게 변경

- entitymanager.flush() 삭제
